osu.Game/Overlays/NewsOverlay.cs

```diff
@@ -67,7 +67,26 @@ private void load()
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            article.BindValueChanged(onArticleChanged, true);
+            article.BindValueChanged(onArticleChanged);
+        }
+
+        private bool displayUpdateRequired = true;
+
+        protected override void PopIn()
+        {
+            base.PopIn();
+
+            if (displayUpdateRequired)
+            {
+                article.TriggerChange();
+                displayUpdateRequired = false;
+            }
+        }
+
+        protected override void PopOutComplete()
+        {
+            base.PopOutComplete();
+            displayUpdateRequired = true;
```

Comment:
this going to trigger a reload every time? is that intentional?

Reply:
Yes, because we always want to see latest news every time we open the overlay

Reply:
We don't do this in other overlays, is there any reason to only do it here? And with no limit on how often? If someone spams the button it's going to send tens of requests per second.

I'd suggest not adding such behaviour until we have a refresh button, or some kind of debounce logic.

Reply:
https://github.com/ppy/osu/blob/8a9036938f30da45e3fa37b3446a636abad45cec/osu.Game/Overlays/DashboardOverlay.cs#L96

You can't spam the button as the overlay has to close completely (i.e. pop out complete).

Reply:
And also we do the same thing for friends overlay

Reply:
Fair enough.